Source/WarpX.H

```diff
@@ -1,12 +1,17 @@
 #ifndef WARPX_H_
 #define WARPX_H_
 
-#include <iostream>
-#include <memory>
-#include <array>
+#include <MultiParticleContainer.H>
+#include <PML.H>
+#include <BoostedFrameDiagnostic.H>
+#include <BilinearFilter.H>
+#include <NCIGodfreyFilter.H>
 
-#ifdef _OPENMP
-#include <omp.h>
+#ifdef WARPX_USE_PSATD
+#   include <SpectralSolver.H>
```

Comment:
I like this indentation, if everyone agrees with it we could probably adopt it more generally and specify it in the `CONTRIBUTING.md`. In particular we have a bunch of nested `#if` that could benefit from that.

Reply:
Thank you for the kind words :blush: 

We tried to find something easily readable in the past and defined [this loose guide](https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md) for ourselves, so it was still in my muscle memory :)

Reply:
Thanks, yes it helps quite a bit with nested `#if`s :blush: